### ISSUE TITLE
Error loading plugin

### ISSUE THREAD

**Issue by VictorEnigma** [kii-chan-reloaded/GeneticChickengineering#2]
>.... [09:32:22 ERROR]: [GeneticChickengineering] path to 'plugins/GeneticChickengineering/GCE.db': '/home/container/plugins/GeneticChickengineering' does not exist
>.... [09:32:22 ERROR]: [GeneticChickengineering] Connection to database failed. Aborting initialization.
>.... [09:32:22 ERROR]: [GeneticChickengineering] Check above for more information about the error.

**Comment by Boomer-1** [comment]
I am having the same issue. when adding the plugin to the plugins folder and starting the server it does not setup a folder for it.

**Comment by kii-chan-reloaded** [comment]
Thank you both for the bug report! I was able to reporduce this on my test server. When adding the database early on in development, I had missed the blurb on [getDataFolder](https://papermc.io/javadocs/paper/1.16/org/bukkit/plugin/Plugin.html#getDataFolder--) where it warns that the folder may not yet exist. I have added a check for a nonexistant folder, and on my test server this resolved the issue. Download and install the latest release to try it out.

I will leave this issue open for the time being just in case I missed something else. Probably tonight I will close it if I haven't heard anything back.
